pycodestyle.py

```diff
@@ -802,6 +802,7 @@ def missing_whitespace_around_operator(logical_line, tokens):
     Okay: alpha[:-i]
 
     E225: i=i+1
+    E225: def foo(a, b)->int:
```

Comment:
Have you tried removing this and running the test suite?

Reply:
Pushed it removed right now to run the full py2 testsuite

Reply:
Ran without it now, the problem as I see it is that the python2 suite still runs `testsuite/python3.py` as can be seen [here](https://travis-ci.org/PyCQA/pycodestyle/jobs/462447069#L511)